include/dlr_common.h

```diff
@@ -42,6 +42,8 @@ typedef struct {
 
 void ListDir(const std::string& dirname, std::vector<std::string>& paths);
 
+std::vector<std::string> split(const char* s, char delimiter);
```

Comment:
Can we use `split` function from `dmlc/common.h` https://github.com/dmlc/dmlc-core/blob/d06aceb92be200c9649258da726be6bb8e4d6556/include/dmlc/common.h#L23

```
namespace dmlc {
inline std::vector<std::string> Split(const std::string& s, char delim) {
```